src/messaging/CHIPReliableMessageConfig.h

```diff
@@ -0,0 +1,125 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2017 Nest Labs, Inc.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file defines the configuration parameters that are required
+ *      for the CHIP Reliable Messaging Protocol.
+ *
+ */
+#ifndef CHIP_RMP_CONFIG_H_
```

Comment:
our current convention is to use `#pragma once` instead of include guards, can you update these, please?

Reply:
done